content/en/docs/releases/bugs/index.md

```diff
@@ -65,16 +69,40 @@ containing:
 
 * Current and previous logs from all Istio components and sidecar
 
-* Istiod logs:
+    * Istiod logs:
+
+        {{< text bash >}}
+        $ kubectl logs -n istio-system -l app=istiod
+        {{< /text >}}
+
+    * Ingress Gateway logs:
+
+        {{< text bash >}}
+        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l istio=ingressgateway -n $ns ; done
```

Comment:
I think this is more complex than it needs to be. The user of these instructions knows where the gateway is deployed, I don’t think we need to loop over all namespaces.

Reply:
Hmm, I wanted that most people could copy-paste and the command without having to modify it.
Originally, I was inclined to use plain istio-system, but thought that might become less common now with the move to injected gateways, so I thought going through all namespaces might be the easiest since some user are even running multiple gateways on multiple namespaces. WDYT?

Reply:
`istio=ingressgateway` is not guaranteed either, its just a label people happen to put on the pod. I would suggest we just ask them to collect the logs of all the gateways, and can provide an example.